### PR TITLE
Remove set url

### PR DESCRIPTION
**Pull request by jacobmealey**
As @emanuele6 mentioned in #201 using url with --set should be removed. This PR has trurl report an error and provides the following message: `--set does not support url, use --redirect instead`

### CODE REVIEW

**Comment by emanuele6** [comment]
I think this patch makes `--set 'url=foo` error when options are parsed, but still allows `--set 'url:=foo'` or `--iterate 'url=...'`.

**Comment by jacobmealey** [comment]
Oh, good catch, I didn't even know `url:=...` was possible. I just pushed another commit. I'm not super happy with it, so if you have suggestions let me know. 

Also, congrats on becoming a _collaborator_!

File: trurl.c [hunk 1]
```diff
@@ -320,6 +322,8 @@ static void iteradd(struct option *o,
 {
   struct curl_slist *n;
   n = curl_slist_append(o->iter_list, iter);
+  if(!strncmp("url", iter, 3))
+    errorf(ERROR_SET, "--iterate does not support url");
```
Comment (bagder) [inline trurl.c:326]: ERROR_ITER ?

File: trurl.c [hunk 2]
```diff
@@ -311,6 +311,8 @@ static void setadd(struct option *o,
 {
   struct curl_slist *n;
   n = curl_slist_append(o->set_list, set);
+  if(!strncmp("url", set, 3))
+    errorf(ERROR_SET, "--set does not support url, use --redirect instead");
```
Comment (bagder) [inline trurl.c:315]: Shouldn't this check be done *before* curl_slist_append() is called?

**Review by bagder — APPROVED** [review]

**Comment by bagder** [comment]
Thanks!

**Pull request closed without merging**